examples/kubernetes/connectivity-check/Makefile

```diff
@@ -30,6 +32,10 @@ $(DEFAULT_OUT): $(SRC)
 	@echo '# Automatically generated by Makefile. DO NOT EDIT' > $(DEFAULT_OUT)
 	$(QUIET)$(CUE) dump > $(DEFAULT_OUT)
 
+$(IPv6_ONLY_OUT): $(SRC)
+	@echo '# Automatically generated by Makefile. DO NOT EDIT' > $(IPv6_ONLY_OUT)
+	$(QUIET)$(CUE) -t ipFamily=IPv6 dump > $(IPv6_ONLY_OUT)
```

Comment:
Sorry, I missed out on the review of Joe's PR on Cue usage in connectivity-checks. But I think this line should append the cue dump `$(QUIET)$(CUE) -t ipFamily=IPv6 dump >> $(IPv6_ONLY_OUT)` to keep the autogenerated header on the top of the file, right?

Reply:
this is very good catch, thanks a lot :100: 

Let me take this chance to update for other targets as well.

Reply:
That would be great. :vulcan_salute: